docs/UserGuide.md

```diff
@@ -208,6 +209,16 @@ Examples:
 * `find Mac` returns `MacDonald`
 * `find fairprice` returns `NTUC Fairprice` <br>
 
+### Viewing a company : `view`
+
+`view` displays the points of contact and transaction log of a company.
```

Comment:
Perhaps could consider replacing it with "Displays the points of contact and transaction log of the specified company."

Reply:
Hi, I've just updated it to what you said